src/app/address/controllers/address.js

```diff
@@ -109,12 +72,12 @@ class AddressController extends BaseController {
       addressLocality,
       addressYearFrom,
     } = undefined,
```

Comment:
I thought it is not good practice to assign `undefined` I would think null would be appropriate